src/DetailsView/components/issues-table.tsx

```diff
@@ -108,14 +110,17 @@ export class IssuesTable extends React.Component<IssuesTableProps> {
 
     private renderComponent(): JSX.Element {
         const cardCount = this.getCardCount();
+        const assessment = this.props.getProvider().forType(this.props.selectedVisualizationType);
```

Comment:
nit: I feel like refactoring this logic out to a separate component like earlier still makes sense but not 100% necessary 100% necessary right now.